### PR TITLE
EARTH-256: Add behat config with updated project.yml

### PR DESCRIPTION
**Pull request by kbrownell**
# NOT READY

# Summary
- Changes to the `blt/project.yml` file should help BLT generate a local.yml file that includes additional paths, like `/modules` and `/profiles`, to look for behat tests.
- It also allows us to output the results as an html page, in addition to the `pretty` progress report.
- We discussed replacing `tests:behat` or creating a new, `sws:behat`. I like creating an `sws:` prefix for BLT commands we override.  That way, if we don't have to wonder, did we end up overriding this?  It's incredibly clear.
- This PR also includes a working .travis.yml file, up to site install due to a lack of ssh keys for pulling from production.

# Urgency
- I have included in this pull request the rc version of caravan, so there is no immediate urgency to close.

# Steps to Test
- Setup requires a bit of work.  Checkout this branch and `run composer update`.
- Place a feature file, somewhere like: `docroot/modules/stanford/stanford_news/tests/stanford_news.feature` from https://raw.githubusercontent.com/SU-SWS/linky_clicky/5.x/sites/uat/features/high-value/stanford_news.feature.
- Run `blt sws:behat` and it should find the stanford_news.feature file.
- You should be able to see output from the tests as they run and afterwards find the full results at http://[local_url]/reports/behat/index.html.

# Affected versions or modules
- https://github.com/SU-SWS/drupal-caravan/releases/tag/1.0.0-rc

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/EARTH-256

### CODE REVIEW

File: blt/project.yml [hunk 1]
```diff
@@ -64,3 +64,11 @@ cm:
     dirs: { sync: { path: '${cm.core.path}/default' } }
   features:
     no-overrides: false
+
+behat:
+  config: '${repo.root}/vendor/su-sws/linky_clicky/behat.blt.yml'
```
Comment (sherakama) [inline blt/project.yml]: Where does this vendor package come from? I see a 6.x branch in github but nothing in this repositories composer.json file nor in packagist.

File: blt/src/Commands/SWSBehatCommand.php [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+/**
+ * Defines commands in the "sws" namespace.
+ */
+class SWSBehatCommand extends BehatCommand {
+  /**
+     * Executes all behat tests.
+     *
+     * @command sws:behat
+     * @description Executes all behat tests. This optionally launch PhantomJS or Selenium prior to execution.
+     * @usage
```
Comment (sherakama) [inline blt/src/Commands/SWSBehatCommand.php]: Any plans on allowing for the calling of different suites? eg dev vs prod?

Comment (sherakama) [inline blt/src/Commands/SWSBehatCommand.php]: Perhaps a `sws:behat:prod` command or a `sws:behat:dev`

Comment (kbrownell) [inline blt/src/Commands/SWSBehatCommand.php]: Plans are beginning, I was trying to punt this to a phase 2 of caravan, focused on automated testing.  But I very much like the idea of being able to append a tag/suite tied to the environment in which the tests will be run.

File: blt/project.yml (hunk 1 above)
Comment (kbrownell) [inline blt/project.yml]: linky_clicky is now pulled in with composer, thanks to this recent merge: https://github.com/SU-SWS/linky_clicky/pull/176.

**Pull request closed without merging**